Source/WebInspectorUI/UserInterface/Views/ContextMenuUtilities.js

```diff
@@ -178,24 +178,28 @@ WI.appendContextMenuItemsForSourceCode = function(contextMenu, sourceCodeOrLocat
 
     if (sourceCode instanceof WI.Resource && !sourceCode.localResourceOverride) {
         if (sourceCode.urlComponents.scheme !== "data") {
+
+            // Information collected about resources loaded before Web Inspector was opened is missing most metadata, like resource timing data, HTTP method, request headers, etc.
+            let isDisabled = !sourceCode.requestSentDate;
```

Comment:
Let's create a `WI.Resource.prototype.hasMetadata` or something and use that instead (and maybe for HAR export too).  I don't think we should propagate `requestSentDate` (and the comment) every time we need to ask "does this `WI.Resource` have the full set of details needed to do the thing).

Reply:
Will do.